client/scss/components/_dropdown.legacy.scss

```diff
@@ -123,6 +123,29 @@
 
         li {
             border-width: 0 0 1px;
+
+            a:focus,
+            button:focus {
+                border: 3px solid #ffbf47;
```

Comment:
It would be good to use the scss variable `$color-focus-outline` here.
(1 styles/focus)